scripts/localnet.sh

```diff
@@ -48,26 +48,26 @@ fi
 
 if [[ $BUILD_BINARY == "1" ]]; then
   echo "*** Building substrate binary..."
-  cargo build --release --features "$FEATURES" --manifest-path "$BASE_DIR/Cargo.toml"
+  cargo build --workspace --profile=production --features "$FEATURES" --manifest-path "$BASE_DIR/Cargo.toml"
```

Comment:
We do not want these for local net builds , as it just makes it alot longer. I think the original release build is sufficient